src/routes/users.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+
+router.get('/', async (req, res) => {
+  try {
```

Comment:
why is this a try catch?

Reply:
It's proper to do since we're serving a file and there could be some random issue that makes it not resolve properly. Kind of like the error checking we do in 392 that is usually pointless but protects against strange issues.